bindings/rust/s2n-tls/src/enums.rs

```diff
@@ -193,3 +193,21 @@ impl From<PeerKeyUpdate> for s2n_peer_key_update::Type {
         }
     }
 }
+
+pub enum SerializationVersion {
+    SerializedConnectionNone,
+    SerializedConnectionV1,
```

Comment:
Since these will probably be called like SerializationVersion::SerializedConnectionNone, should the name just be "None" / "V1" so that it's SerializationVersion::None and SerializationVersion::V1 instead?

Reply:
Oh and you need to mark this non_exhaustive

Reply:
Probably also all the other standard derives (#[derive(Debug, PartialEq, Copy, Clone)]) so that it's usable.